shared/css/user-progress.scss

```diff
@@ -1,154 +0,0 @@
-@import 'color';
-@import 'font';
-
-.level_link, .level_link:link, .level_link:visited {
-  box-sizing: content-box;
-  border-radius: 20px;
-  min-width: 22px;
-  line-height: 22px;
-  padding: 4px;
-  font-size: 16px;
-  text-align: center;
-  display: inline-block;
-  font-family: $gotham-regular;
-  font-weight: normal;
-  color: white;
-  border: 2px solid $lighter_gray;
-
-
-  $transition: background-color .2s ease-out, border-color .2s ease-out, color .2s ease-out;
-  transition: $transition;
-  -moz-transition: $transition;
-  -webkit-transition: $transition;
-  -o-transition: $transition;
-}
-
-.level_link.not_tried, .level_link.not_tried:link, .level_link.not_tried:visited {
-  background-color: $level_not_tried;
-  color: $charcoal !important;
-}
-
-.level_link.locked, .level_link.locked:link, .level_link.locked:visited {
-  background-color: $level_not_tried;
-  color: $charcoal !important;
-}
-
-a.level_link.not_tried:hover,
-a.level_link.attempted:hover,
-a.level_link.perfect:hover,
-a.level_link.passed:hover,
-a.level_link.submitted:hover
- {
-    background-color: $level_current;
-    color: white;
-    text-decoration: none;
-}
-
-.level_link.attempted, .level_link.attempted:link, .level_link.attempted:visited {
-  background-color: $level_attempted;
-  color: $charcoal !important;
-}
-
-.level_link.perfect, .level_link.perfect:link, .level_link.perfect:visited {
-  background-color: $level_perfect;
-  color: white !important;
-}
-
-
-.level_link.passed, .level_link.passed:link, .level_link.passed:visited {
-  background-color: $level_passed;
-  color: white !important;
-}
-
-.level_link.submitted, .level_link.submitted:link, .level_link.submitted:visited {
-  background-color: $level_submitted;
-  color: white !important;
-}
-
-.level_link.review_accepted, .level_link.review_accepted:link, .level_link.review_accepted:visited {
-  background-color: $level_review_accepted;
-  color: white !important;
-}
-
-.level_link.review_rejected, .level_link.review_rejected:link, .level_link.review_rejected:visited {
-  background-color: $level_review_rejected;
-  color: white !important;
-}
-
-.user-stats-block
```

Comment:
this is the only one that I could find which was actually getting applied to any elements in our product. I checked the script overview page and the MiniView, and they appeared to render the same with or without these styles.